packages/sdk-tizen/src/deviceManager.ts

```diff
@@ -71,44 +72,43 @@ const formatXMLObject = (
     return {};
 };
 
-export const launchTizenEmulator = async (name: string | true): Promise<boolean> => {
+export const launchTizenEmulator = async (name: string | true, hideDevices?: boolean): Promise<boolean> => {
```

Comment:
This method should only run/show emulators. Does it need to have that new hideDevices prompt?

Reply:
Not sure why its only for emulators? The name can be changed.